### ISSUE TITLE
Not showing WinMD Volume

### ISSUE THREAD

**Issue by HTWingNut** [Endracion/mdadm-raid-mount-windows#1]
Thank you for the great write-up and this driver/app for mdadm for Windows. However, I have a Seagate Exos 2x18 and followed your instructions, made the partitions, formatted NTFS. installed the inf driver in Windows, plugged in the disk. The disk and two partitions show up in disk management. WinMD Controller shows up under Storage Volumes in Device Manager, but no WinMD Volume. I am connected by SATA. 

Any suggestions? Thanks.

**Comment by Endracion** [comment]
Hey! Since I didn't create any of the tools, I'm not sure I will be of much help, but I can say that when I first did this, it did not work either. I simple started again from scratch, making sure that I erase everything to do with the raid beforehand. You can follow the erasure method explained here: https://www.digitalocean.com/community/tutorials/how-to-create-raid-arrays-with-mdadm-on-ubuntu-22-04#resetting-existing-raid-devices-optional
On my second try, doing the same thing I did before, it worked.

**Comment by dawger1** [comment]
I have the same issue. Followed the steps, installed winMD, and can see the drive and partitions in disk management, but don’t have a WinMD volume listed in device manager, and cannot see the drive in explorer.  I wiped the drive and started over from scratch, with the exact same result.  

I’ve spent a couple of hours on this already. I might just give up and use the drive as a normal external hard drive. Not sure that the juice is worth the squeeze

**Comment by Endracion** [comment]
Hm, well that's unfortunate, I wonder if I missed a step somewhere, but I really can't recall. I'll look into it and update if I find anything. Sorry that it's not working for you guys.

**Comment by dawger1** [comment]
Thank you so much for trying, and for the detailed write up!  I wouldn’t have even thought to try if it weren’t for you!

**Comment by HTWingNut** [comment]
I ended up getting it to work. I wiped the disk with Windows diskpart clean command and started over. To do this open windows elevated command prompt (administrator):

```
diskpart
list disk
select disk 2 (choose proper disk number for your drive)
clean
```

Be sure to choose proper disk number because it will wipe all partition information immediately without remorse.

I then removed the disk from my windows machine, installed the winmd driver (right click inf - install) and rebooted.

Then installed the disk in my Linux box, and from Linux I used:

```
parted /dev/sda mklabel gpt
parted --align optimal /dev/sda mkpart primary 0% 50%
parted --align optimal /dev/sda mkpart primary 50% 100%
mdadm --create --verbose /dev/md0 --level=stripe --raid-devices=2 /dev/sda1 /dev/sda2
mkntfs -Q /dev/md0
```

All this with `sudo` command and also make sure to swap proper drive letter from `/dev/sda` to whatever your disk letter is in Linux.

I then plunked the disk back into my windows box, booted up, and it was detected. It auto assigns a drive letter and couldn't figure a way to change it.

**Comment by Endracion** [comment]
Great to hear you managed it another way! I'll add your solution to the guide if you don't mind?

To change the drive letter, the original driver creator only offered a solution through the registry:  
maharmstone/winmd#4

**Comment by HTWingNut** [comment]
Yeah, sure add whatever you find helpful. Thanks for tip on changing drive letter. I can't say I'm not disappointed with the lackluster performance. I guess I was expecting better. It would be nice if Seagate offered a tool to manage this in Windows at least.

**Comment by empereurfn** [comment]
Can I ask what speeds you guys are getting? I am only able to get 410mbs read and 120mbs write on mach.2 Seagate drive

**Comment by Endracion** [comment]
> Can I ask what speeds you guys are getting? I am only able to get 410mbs read and 120mbs write on mach.2 Seagate drive

Speeds are listed in the main guide, not sure what you're testing with, but just your write is kind of slow compared to my crystal disk mark results.

**Comment by Surena0647** [comment]
> I ended up getting it to work. I wiped the disk with Windows diskpart clean command and started over. To do this open windows elevated command prompt (administrator):
> 
> ```
> diskpart
> list disk
> select disk 2 (choose proper disk number for your drive)
> clean
> ```
> 
> Be sure to choose proper disk number because it will wipe all partition information immediately without remorse.
> 
> I then removed the disk from my windows machine, installed the winmd driver (right click inf - install) and rebooted.
> 
> Then installed the disk in my Linux box, and from Linux I used:
> 
> ```
> parted /dev/sda mklabel gpt
> parted --align optimal /dev/sda mkpart primary 0% 50%
> parted --align optimal /dev/sda mkpart primary 50% 100%
> mdadm --create --verbose /dev/md0 --level=stripe --raid-devices=2 /dev/sda1 /dev/sda2
> mkntfs -Q /dev/md0
> ```
> 
> All this with `sudo` command and also make sure to swap proper drive letter from `/dev/sda` to whatever your disk letter is in Linux.
> 
> I then plunked the disk back into my windows box, booted up, and it was detected. It auto assigns a drive letter and couldn't figure a way to change it.

Hello there,

I just purchased a Mach.2 2x18 drive and I have followed your steps as well and I still dont see anything in windows. The drive shows up in disk management but cant otherwise interact with it except change it to dynamic. In device manager I don't see anything regarding WinMD, while I did install it, and again just incase. In task manager the drive also shows up, but it says Formatted: 0MB, which does not make sense either.

I guess I'm stuck and cant stripe the drives(?)
